### ISSUE TITLE
Indented regions in docstrings are highlighted as code

### ISSUE THREAD

**Issue by memeplex** [microsoft/pylance-release#83]
## Environment data

-   Language Server version: 2020.7.0
-   OS and version: Ubuntu 20.04
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.2

## Expected behaviour

Python highlighting isn't applied to natural language. 

## Actual behaviour

Lots of usual words (and, or, is, etc) are colorized in docstrings.

**Comment by jakebailey** [comment]
Can you please provide an example? This may be a case where the code is indented and being treated as a code block, similar to: #48

**Comment by memeplex** [comment]
I don't have the required environment just right now, but AFAICR `argparse.Action` was one case.

**Comment by huguesv** [comment]
`argparse.Action.__doc__` is all indented, so it is causing it to be treated as code.

```
Information about how to convert command line strings to Python objects.

    Action objects are used by an ArgumentParser to represent the information
    needed to parse a single argument from one or more strings from the
    command line. The keyword arguments to the Action constructor are also
    all attributes of Action instances.

    Keyword Arguments:

        - option_strings -- A list of command-line option strings which
            should be associated with this action.

        - dest -- The name of the attribute to hold the created object(s)

        - nargs -- The number of command-line arguments that should be
            consumed. By default, one argument will be consumed and a single
            value will be produced.  Other values include:
                - N (an integer) consumes N arguments (and produces a list)
                - '?' consumes zero or one arguments
                - '*' consumes zero or more arguments (and produces a list)
                - '+' consumes one or more arguments (and produces a list)
            Note that the difference between the default and nargs=1 is that
            with the default, a single value will be produced, while with
            nargs=1, a list containing a single value will be produced.

        - const -- The value to be produced if the option is specified and the
            option uses an action that takes no values.

        - default -- The value to be produced if the option is not specified.

        - type -- A callable that accepts a single string argument, and
            returns the converted value.  The standard Python types str, int,
            float, and complex are useful examples of such callables.  If None,
            str is used.

        - choices -- A container of values that should be allowed. If not None,
            after a command-line argument has been converted to the appropriate
            type, an exception will be raised if it is not a member of this
            collection.

        - required -- True if the action must always be specified at the
            command line. This is only meaningful for optional command-line
            arguments.

        - help -- The help string describing the argument.

        - metavar -- The name to be used for the option's argument with the
            help string. If None, the 'dest' value will be used as the name.
    ```

**Comment by memeplex** [comment]
There are many many docstrings with indented natural language, please see my comment https://github.com/microsoft/pylance-release/issues/41#issuecomment-656375434.

**Comment by bradennapier** [comment]
![image](https://user-images.githubusercontent.com/15365418/89723843-7dc5f280-d9c9-11ea-9fef-2fe708a39753.png)

**Comment by jakebailey** [comment]
The next release includes a few docstring improvements which should resolve this for a few different docstring styles, including numpy/google/sphinx styles.

**Comment by jakebailey** [comment]
Just to give a preview ahead of tomorrow, this is `argparse.Action`:

![image](https://user-images.githubusercontent.com/5341706/106539041-125d6900-64b2-11eb-9e22-df3a4f9a93b2.png)

`hmac.new`:

![image](https://user-images.githubusercontent.com/5341706/106539099-30c36480-64b2-11eb-992d-707ad2b8c80d.png)

And a few others:

![image](https://user-images.githubusercontent.com/5341706/106539119-3e78ea00-64b2-11eb-8ec0-2ed38a2fbc00.png)

![image](https://user-images.githubusercontent.com/5341706/106539174-581a3180-64b2-11eb-988c-0c8f47683de9.png)

(@bschnurr deserves all of the credit for this 😄)

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.2.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202120-3-february-2021
